### PR TITLE
LLP-24-SupportnComplaintsPage

### PR DESCRIPTION
**Pull request by mdainys**

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import styles from "@/styles/Form.module.scss"
+
+function Form(props) {
+
```
Comment (pinenetcoder) [inline src/components/Form.js:4]: Make a name of file and component ContactForm

File: src/styles/Form.module.scss [hunk 2]
```diff
@@ -0,0 +1,62 @@
+
+.formInputBlock {
+   width: 468px;
+   display: flex;
+   flex-direction: column;
+
+
```
Comment (pinenetcoder) [inline src/styles/Form.module.scss:7]: delete xtra spaces

File: src/components/Form.js (hunk 1 above)
Comment (mdainys) [reply src/components/Form.js:4]: Fixed.

File: src/styles/Form.module.scss (hunk 2 above)
Comment (mdainys) [reply src/styles/Form.module.scss:7]: Fixed.

**Pull request merged**